packages/block-library/src/query/edit/inspector-controls/index.js

```diff
@@ -173,19 +174,22 @@ export default function QueryInspectorControls( props ) {
 		showParentControl ||
 		showFormatControl;
 	const dropdownMenuProps = useToolsPanelDropdownMenuProps();
-
 	const showPostCountControl = isControlAllowed(
 		allowedControls,
 		'postCount'
 	);
 	const showOffSetControl = isControlAllowed( allowedControls, 'offset' );
 	const showPagesControl = isControlAllowed( allowedControls, 'pages' );
-
 	const showDisplayPanel =
 		showPostCountControl || showOffSetControl || showPagesControl;
 
 	return (
 		<>
+			{ !! postType && (
+				<BlockInfo>
+					<CreateNewPostLink postType={ postType } />
+				</BlockInfo>
+			) }
```

Comment:
This link and the `BlockInfo` slot was just removed in https://github.com/WordPress/gutenberg/pull/67189. I take it this was just a leftover from the last rebase?

Reply:
yeah probably 😄 I probably didn't fetch before. Thanks for spotting